--- amdgpu_pm-stock.c	2026-08-20 09:32:55.747100 -0400
+++ amdgpu_pm.c	2026-08-20 09:32:55.747100 -0400
@@ -3333,6 +3333,9 @@
 					 struct device_attribute *attr,
 					 char *buf)
 {
+	if (amdgpu_ignore_pcap)
+		return sysfs_emit(buf, "%i\n", 0);
+
 	return amdgpu_hwmon_show_power_cap_generic(dev, attr, buf, PP_PWR_LIMIT_MIN);
 }
 
@@ -3340,6 +3343,9 @@
 					 struct device_attribute *attr,
 					 char *buf)
 {
+	if (amdgpu_ignore_pcap)
+		return sysfs_emit(buf, "%i\n", 500 * 1000000);
+
 	return amdgpu_hwmon_show_power_cap_generic(dev, attr, buf, PP_PWR_LIMIT_MAX);
 
 }
